test/TensorFlow/graph_op_inst.sil

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-sil-opt -sil-print-all %s | %FileCheck %s
+
+sil_stage raw
+
+import Builtin
+import Swift
+
+// Dummy struct mimicking the real `Tensor` type.
+struct Tensor<Scalar> {}
+
+sil @attribute_test : $@convention(thin) () -> Tensor<Float> {
+bb0:
+  %0 = graph_op "tf.Dummy"() [int 1, float 3.14, metatype $Float, array [[1, 2], [3, 4]]] : $Tensor<Float>
+  return %0 : $Tensor<Float>
+}
+
+// CHECK-LABEL: sil @attribute_test : $@convention(thin) () -> Tensor<Float> {
+// CHECK: bb0:
+// CHECK-NEXT:   %0 = graph_op "tf.Dummy"() [int 1, float 3.14{{[0-9]*}}, metatype $Float, array {{\[\[}}1, 2], [3, 4]]] : $Tensor<Float>
+// CHECK-NEXT:   return %0 : $Tensor<Float>
+// CHECK-NEXT: }
+
+sil @chained_op_test : $@convention(thin) (Tensor<Float>, Tensor<Float>) -> Tensor<Float> {
+bb0(%0 : $Tensor<Float>, %1 : $Tensor<Float>):
+  %2 = graph_op "tf.Add"(%0 : $Tensor<Float>, %1 : $Tensor<Float>) [T $Float] : $Tensor<Float>
```

Comment:
I'd suggest syntactically making the attributes another "parameter list" which is only printed if attributes are present.  The example above would be:

%2 = graph_op "tf.Add"(%0 : $Tensor<Float>, %1 : $Tensor<Float>) (T: $Float) : $Tensor<Float>

And if there are no attributes it would be omitted:

%2 = graph_op "tf.Add"(%0 : $Tensor<Float>, %1 : $Tensor<Float>) : $Tensor<Float>

WDYT?

Reply:
Colons are going to be really confusing because the attribute value is not a type signature.

Reply:
Plus, I think since `[]` is already used for instruction attributes, we can make it continue to carry that meaning.

Reply:
I am a fan of the current syntax using `[]`.

- There is no colon between attribute names and values:
  - `[name1 1, name2 2.0, name3 $Float]`.
- If there are no attributes, the `[]` is not printed.

Reply:
As mentioned in another comment, we also have to add a (weird) type system for these constants as well.  I think that argues for using equals or something to distinguish these from normal arguments, e.g. something like:

[name1 = i64 1, name2 = f64 2.0, name3 = $Float, name4 = [[i32 1, i32 2, i32 3], f64 17.0]]

I still don't love the outer square brackets tho.

Reply:
I handled integer/floating point datatypes, please re-review.
I'm not sure I parsed datatypes (`i32`, `f64`) in the optimal way, would appreciate your feedback.

I also added a `=` to separate attribute names from values: `<name> = <value>`.